scopes/explorer/ui/component-card/component-card.tsx

```diff
@@ -62,10 +64,15 @@ export function ComponentCard({
   envIcon,
   isDeprecated = false,
   href,
+  overrideInternalLink,
```

Comment:
isn't this.. just href? 🤔 
just use it!

Reply:
if I use the href it makes the link external

Reply:
pass `exernal` flag like we discussed
make it false by default, and apply where relevant